spec/mailers/previews/email.html.erb

```diff
@@ -0,0 +1,170 @@
+<!--from https://github.com/rails/rails/blob/6-0-stable/railties/lib/rails/templates/rails/mailers/email.html.erb-->
+
+<!DOCTYPE html>
+<html><head>
+  <meta name="viewport" content="width=device-width" />
+  <style type="text/css">
+    html, body, iframe {
+      height: 100%;
+    }
+
+    body {
+      margin: 0;
+    }
+
+    header {
+      width: 100%;
+      padding: 10px 0 0 0;
+      margin: 0;
+      background: white;
+      font: 12px "Lucida Grande", sans-serif;
+      border-bottom: 1px solid #dedede;
+      overflow: hidden;
+    }
+
+    dl {
+      margin: 0 0 10px 0;
+      padding: 0;
+    }
+
+    dt {
+      width: 120px;
+      padding: 1px;
+      float: left;
+      clear: left;
+      text-align: right;
+      color: #7f7f7f;
+    }
+
+    dd {
+      margin-left: 90px; /* 80px + 10px */
```

Comment:
```suggestion
      margin-left: 125px; /* 120px + 5px */
```

Reply:
Or leave as is if you decide to accept the other recommendation which combines the two new fields under a single ‘Template’ key

Reply:
I've accepted the other change and left this one.